src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx

```diff
@@ -1,44 +1,28 @@
 import { Select, Space, Typography } from "antd";
 import React from "react";
 import { useDispatch, useSelector } from "react-redux";
-import {
-  useGetPotentialProjectsToShareToQuery
-} from "../../../apis/projects/projects";
+import { useGetPotentialProjectsToShareToQuery } from "../../../apis/projects/projects";
```

Comment:
Don't think this is being used and can be removed?

Reply:
Removed in 6d21c776a1f9698293d9d200534ff09576a74775